projectroles/tests/test_views.py

```diff
@@ -1968,6 +1968,16 @@ def setUp(self):
             project=self.project,
         )
 
+        # Init project_star setting
```

Comment:
To be honest I do not understan, why this get/post view tests are working when I add this setting and do not without it. The probem went deeply to AppSettingAPI methods, where I was little bit lost. So it is working as it is, but I do not know why(

Reply:
Can you clarify on what exactly was not working and how?

Reply:
If I delete this project_star setting in test, than I become failure in requests:
```response = self.client.get(
              reverse(
                  'projectroles:update',
                  kwargs={'project': self.project.sodar_uuid},
              )
          )
```
The exception is being made in `app_settings.py` file in:
```raise KeyError(
            'Setting "{}" not found in app plugin "{}"'.format(
                setting_name, app_name
            )
        )
```
I am just confused, is it looking for every app_setting for projectroles?